src/ui/public/chrome/directives/app_switcher/app_switcher.less

```diff
@@ -102,10 +102,9 @@ body { overflow-x: hidden; }
       height: @app-icon-height;
       width: @as-closed-width;
 
-      > img {
+      > .image {
```

Comment:
we want to apply this stile to the image as well as to the svg if image was replaced, so i am switching from element selector to class selector